docs/content/technical-specifications.md

```diff
@@ -23,6 +24,7 @@ We explicitly test the NGINX Ingress Controller (NIC) on a range of Kubernetes p
 {{% table %}}
 | NIC Version | Supported Kubernetes Version | NIC Helm Chart Version | NIC Operator Version | NGINX / NGINX Plus version |
 | --- | --- | --- | --- | --- |
+| 3.1.0 | 1.26 - 1.22 | 0.16.3 | 1.3.0 | 1.23.3 / R28 |
```

Comment:
I am anticipating NIC Version change from `3.0.2` to `3.1.0` here.

I committed NIC Helm Chart Version increase from `0.16.2` to `0.16.3` as it's on the `0.x` series -- is this okay here? Or should I bump it up to `0.17.0` in this commit as well?

Reply:
The versioning related things will all be handled by the release processes.
you don't need to update these as part of your PR.

Reply:
Latest commit removes the manual addition.